packages/app/src/app/components/Preview/DevTools/Tabs/Tab/elements.ts

```diff
@@ -33,9 +33,9 @@ export const Tab = styled.div<{ active: boolean; isOver: boolean }>`
   ${props =>
     props.active
       ? css`
-          background-color: ${props.theme['tab.activeBackground'] ||
-            props.theme['editor.background'] ||
-            'transparent'};
+          background-color: ${props.theme['input.background'] ||
```

Comment:
input background instead of tab background? america explain

Reply:
A lot of them don't have tab background so that was ruining danny's intentions :(

Reply:
should we pick tab.activeBackground and then fallback to input.background?

Reply:
But the colors aren't the same, it looks off

Reply:
Hmm, what about these tabs? The browser looks nicer, but the other tabs are now inconsistent. I'm thinking this might be something the theme should solve (different colors).

![image](https://user-images.githubusercontent.com/587016/70919599-6c3a6f80-2021-11ea-9219-fa5297809888.png)

Reply:
Maybe we could use the old colors for the current theme still? Then when we switch to CodeSandbox Black everything should look like the current design cc @DannyRuchtie

Reply:
I also like @siddharthkp suggestion of defaulting to `tab.activeBackground` and fallback to `input.background`, but with mixed feelings. Remember that these themes probably decided to no set the `tab.activeBackground` on purpose.

Reply:
@CompuIves we moved the conversation to slack, so you're missing some context now :)

Reply:
> Remember that these themes probably decided to no set the tab.activeBackground on purpose.

oh p.s. vscode falls back for you  if you don't set `tab.activeBackground`, based on its own dark/light rules. So we can also do the same based on what default looks best on codesandbox

Reply:
> oh p.s. vscode falls back for you if you don't set tab.activeBackground, based on its own dark/light rules. So we can also do the same based on what default looks best on codesandbox

Yes, agree. We apply this too since we use the theme generated by vscode AFAIK. Need to double check, I've been wrong before.